lib/dataElementListener.js

```diff
@@ -62,9 +64,16 @@ class DataElementImporter extends SHRDataElementParserListener {
     }
   }
 
+
   enterDoc(ctx) {
     // Process the namespace
-    const ns = ctx.docHeader().namespace().getText();
+    let ns = '';
+    if (ctx.docHeader().namespace()) {
+      ns = ctx.docHeader().namespace().getText();
+    }
+    else {
+      logger.error('Namespace declaration not found. ERROR_CODE:11???');
```

Comment:
This needs a real error code. Just try to find the next number that's not used.  This page may help, although I'm not sure if it is up to date: https://github.com/standardhealth/shr-cli/wiki/Error-Message-Documentation#all-error-codes